src/java/com/wolfssl/WolfSSL.java

```diff
@@ -905,6 +905,12 @@ public static native int getPkcs8TraditionalOffset(byte[] in, long idx,
      */
     public static native String[] getProtocols();
 
+    /**
+     * Checks which protocols where built into wolfSSL with Mask
+     *
+     * @return an array of Strings for supported protocols
+     */
+    public static native String[] getProtocolsMask(long mask);
```

Comment:
We need to add an @param notation in the comments for Javadoc.  "ant" fails on OSX with the following message:

```
  [javadoc] /Users/chrisc/Desktop/kojo/wolfssljni/src/java/com/wolfssl/WolfSSL.java:913: warning: no @param for mask
  [javadoc]     public static native String[] getProtocolsMask(long mask);
```